app/components/zitting-text-document-container.hbs

```diff
@@ -0,0 +1,59 @@
+
+<AuPill @icon="info-circle" @iconAlignment="left" class="au-u-margin-top-small">
```

Comment:
I would use the same structure as [the other sections of the page](https://github.com/lblod/frontend-gelinkt-notuleren/blob/4a6784c5217f8de2fac8462ae1a3cd0f1481d615/app/components/meeting-form.hbs#L91). It looks a bit different now.

Reply:
well I mean it is in the feature passport

Reply:
Yea but it doesn't look the same as the others on the same page. They use a different class / html structure, right? Not the `AuPill` component.

Reply:
Screenshot to make it clearer what I mean:

![image](https://user-images.githubusercontent.com/3533236/117428133-102dff00-af26-11eb-9713-ea055ff65b67.png)

Reply:
The passport version looks more like the first one.

Reply:
ah fair enough